### DISCUSSION TITLE
How to run uc mode on colab

### DISCUSSION THREAD

**Discussion by parth76** [seleniumbase/SeleniumBase#2802] [Q&A] [answered]
when I run seleniumbase without uc mode it runs fine
![example_1](https://github.com/seleniumbase/SeleniumBase/assets/56641757/b9a2464f-0e90-446a-9716-d70bb959a2c5)

but when I run it with uc mode it gives me this error
![example_2](https://github.com/seleniumbase/SeleniumBase/assets/56641757/e91e153a-a4d0-4617-b37a-e49864b8211e)

**Comment by mdmintz** [comment, marked as answer]
Duplicate of https://github.com/seleniumbase/SeleniumBase/issues/2778#issuecomment-2115245835

**Reply by parth76** [reply]
Thank you Sir, for your reply I tried setting the binary path of the chrome browser with the below code but it still does not work, I also tried virtual display.
```
from seleniumbase import Driver,SB
from sbvirtualdisplay import Display

with SB(binary_location = '/usr/bin/chromium-browser',uc = True, xvfb = True, headless = False) as sb:
  sb.get('https://httpbin.org/ip')
```
it gives this error 
```
WebDriverException: Message: unknown error: cannot connect to chrome at 127.0.0.1:9222
from chrome not reachable
```
Same code works fine without uc mode

**Reply by mdmintz** [reply]
You have to use regular chrome, not chromium. Binary would be called google-chrome or google-chrome-stable.

**Reply by parth76** [reply]
Thanks it solved the problem, You really helped me.
